### PR TITLE
Fix lint issues in the repo.

### PR DESCRIPTION
**Pull request by michelle192837**
/lint

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -64,14 +64,14 @@ func (e MissingEntityError) Error() string {
 	return fmt.Sprintf("could not find the referenced (%s) %s", e.Entity, e.Name)
 }
 
-// ConfigError is an error for invalid configuration that includes what entity errored.
-type ConfigError struct {
+// Error is an error for invalid configuration that includes what entity errored.
+type Error struct {
```
Comment (chases2) [inline config/config.go:68]: I think "ConfigError" or "EntityError" is better. Easy to typo or confuse `error` with `Error`

**Review by chases2 — COMMENTED** [review]
Thanks for this! Only one comment

File: config/config.go (hunk 1 above)
Comment (fejta) [reply config/config.go:68]: Issue is that this gets written as `config.ConfigError` outside the package, which stutters, and why the linter will complain about it.

Comment (chases2) [reply config/config.go:68]: That's a good point. "ValidationError"?

Comment (michelle192837) [reply config/config.go]: ValidationError SGTM, updated!

**Review by chases2 — APPROVED** [review]

**Pull request merged**